smpe/smpe_workflow.xml

```diff
@@ -0,0 +1,1785 @@
+//* -- SPLIT HERE IF NEEDED --
+//*
+//* PRIME CSI
+//*
+//INITCSI  EXEC PGM=IDCAMS,REGION=0M,COND=(4,LT)
+//SMPCSI   DD DISP=OLD,DSN=&amp;CSIHLQ..CSI
+//ZPOOL    DD DISP=SHR,DSN=SYS1.MACLIB(GIMZPOOL)
+//SYSPRINT DD SYSOUT=*
+//SYSIN    DD *
+  REPRO INFILE(ZPOOL) OUTFILE(SMPCSI)
+//*
+//* ALLOCATE SMP/E WORK DATA SETS
+//* DEFINE TARGET &amp; DISTRIBUTION ZONES, OPTIONS, UTILITIES AND DDDEFS
+//*
+//ZONING   EXEC PGM=GIMSMP,REGION=0M,COND=(4,LT)
+//SMPCSI   DD DISP=OLD,DSN=&amp;CSIHLQ..CSI
+//*
+//SMPLOG   DD DSN=&amp;DSPREFIX..SMPLOG,
+//            DISP=(NEW,CATLG,DELETE),
+//            DSORG=PS,
+//            RECFM=VB,
+//            LRECL=3200,
+//            BLKSIZE=0,
+//            UNIT=SYSALLDA,
+#if( $csivol and $csivol != &quot;&quot; and $csivol != '#csivol')
+//            VOL=SER=&amp;CSIVOL,
+#end
+#if( $storclass and $storclass != &quot;&quot; and $storclass != '#storclass')
+//            STORCLAS=&amp;STCLAS,
+#end
+#if( $dataclass and $dataclass != &quot;&quot; and $dataclass != '#dataclass')
+//            DATACLAS=&amp;DTCLAS,
+#end
+#if( $mgmtclass and $mgmtclass != &quot;&quot; and $mgmtclass != '#mgmtclass')
+//            MGMTCLAS=&amp;MGCLAS,
+#end
+//            SPACE=(TRK,(30,15))
+//*
+//SMPLOGA  DD DSN=&amp;DSPREFIX..SMPLOGA,
+//            DISP=(NEW,CATLG,DELETE),
+//            DSORG=PS,
+//            RECFM=VB,
+//            LRECL=3200,
+//            BLKSIZE=0,
+//            UNIT=SYSALLDA,
+#if( $csivol and $csivol != &quot;&quot; and $csivol != '#csivol')
+//            VOL=SER=&amp;CSIVOL,
+#end
+#if( $storclass and $storclass != &quot;&quot; and $storclass != '#storclass')
+//            STORCLAS=&amp;STCLAS,
+#end
+#if( $dataclass and $dataclass != &quot;&quot; and $dataclass != '#dataclass')
+//            DATACLAS=&amp;DTCLAS,
+#end
+#if( $mgmtclass and $mgmtclass != &quot;&quot; and $mgmtclass != '#mgmtclass')
+//            MGMTCLAS=&amp;MGCLAS,
+#end
+//            SPACE=(TRK,(30,15))
+//*
+//SMPLTS   DD DSN=&amp;DSPREFIX..SMPLTS,
+//            DISP=(NEW,CATLG,DELETE),
+//            DSNTYPE=LIBRARY,
+//            RECFM=U,
+//            LRECL=0,
+//            BLKSIZE=32760,
+//            UNIT=SYSALLDA,
+#if( $csivol and $csivol != &quot;&quot; and $csivol != '#csivol')
+//            VOL=SER=&amp;CSIVOL,
+#end
+#if( $storclass and $storclass != &quot;&quot; and $storclass != '#storclass')
+//            STORCLAS=&amp;STCLAS,
+#end
+#if( $dataclass and $dataclass != &quot;&quot; and $dataclass != '#dataclass')
+//            DATACLAS=&amp;DTCLAS,
+#end
+#if( $mgmtclass and $mgmtclass != &quot;&quot; and $mgmtclass != '#mgmtclass')
+//            MGMTCLAS=&amp;MGCLAS,
+#end
+//            SPACE=(TRK,(30,30,80))
+//*
+//SMPMTS   DD DSN=&amp;DSPREFIX..SMPMTS,
+//            DISP=(NEW,CATLG,DELETE),
+//            DSORG=PO,
+//            RECFM=FB,
+//            LRECL=80,
+//            BLKSIZE=0,
+//            UNIT=SYSALLDA,
+#if( $csivol and $csivol != &quot;&quot; and $csivol != '#csivol')

```

Comment:
not sure right now how we can rhyme this if logic with shipping a sample that has has VOL=SER available as comment. (Same is true for the SMS class definitions.)

Reply:
I believe this condition is consistent with the description of the commented statement in the sample JCL.